en/micro-integrator/docs/setup/security/securing_management_api.md

```diff
@@ -100,15 +100,15 @@ the `MI_HOME/conf/` directory) and add the following:
 
 ```toml
 [[management_api.authorization_handler.resources]]
-path = "/user"
+path = "/users"
```

Comment:
/users is authorized by default. This is for other resources other than /users ( line 98 ).